src/libs/ReportActionsUtils.ts

```diff
@@ -324,7 +328,16 @@ function shouldReportActionBeVisible(reportAction: OnyxEntry<ReportAction>, key:
 
     // Ignore closed action here since we're already displaying a footer that explains why the report was closed
     if (reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.CLOSED) {
-        return false;
+        if (!allReports) {
+            return false;
+        }
+        const expenseReport = allReports[reportAction?.originalMessage?.IOUReportID ?? 0] ?? null;
```

Comment:
A bit hard to parse with all the `?`  in it.

Going to take a leap and say that if we have no IOUReportID at this stage then something went wrong. And another leap that if the expense report doesn't exist then another thing went wrong.